tests/async/test_assertions.py

```diff
@@ -658,3 +658,16 @@ async def test_should_print_response_with_text_content_type_if_to_be_ok_fails(
     assert "← 404 Not Found" in error_message
     assert "Response Text:" not in error_message
     assert "Image content type error" not in error_message
+
+
+async def test_should_print_users_message_for_page_based_assertion(
+    page: Page, server: Server
+) -> None:
+    await page.goto(server.EMPTY_PAGE)
+    await page.set_content("<title>new title</title>")
+    with pytest.raises(AssertionError) as excinfo:
+        await expect(page, "Title is not new").to_have_title("old title")
```

Comment:
```suggestion
        await expect(page, "Title is not new").to_have_title("old title", timeout=100)
```

this makes the test pass faster.